README.md

```diff
@@ -13,7 +13,7 @@
 for machine learning. It has a comprehensive, flexible ecosystem of
 [tools](https://www.tensorflow.org/resources/tools),
 [libraries](https://www.tensorflow.org/resources/libraries-extensions), and
-[community](https://www.tensorflow.org/community) resources that lets
+[community](https://www.tensorflow.org/community) resources that let
 researchers push the state-of-the-art in ML and developers easily build and
```

Comment:
It should be 'resources that **let** **researchers**' instead of 'resources that **lets** **researchers**'